openmetadata-service/src/main/java/org/openmetadata/service/util/UserUtil.java

```diff
@@ -254,4 +260,16 @@ public static List<EntityReference> getRoleForBot(String botName) {
     }
     return listOf(RoleResource.getRole(botRole));
   }
+
+  public static void cleanUpBasicAuth() {
+    // Delete Admin UserName
+    UserRepository userRepository = (UserRepository) Entity.getEntityRepository(Entity.USER);
+    try {
+      // Fetch Original User, is available
+      User originalUser = userRepository.getByName(null, ADMIN_USER_NAME, new EntityUtil.Fields(List.of("id,name")));
+      userRepository.delete(ADMIN_USER_NAME, originalUser.getId(), true, true);
```

Comment:
this is problematic if the admin user created a service. We need to clean up these relationships as well. Same goes when we delete the user we should just clean up the relations and keep the entities such user owns and follows should go away if the user is getting hard deleted.

Reply:
fixed it ...instead of deletion we use an empty auth mechanism for non SSO and basic auth will have valid populated auth mechanism